hummingbot/connector/exchange/bittrex/bittrex_api_user_stream_data_source.py

```diff
@@ -127,18 +126,19 @@ async def listen_for_user_stream(self, ev_loop: asyncio.BaseEventLoop, output: a
 
                 async for raw_message in self._socket_user_stream(self._websocket_connection):
                     decode: Dict[str, Any] = self._transform_raw_message(raw_message)
+                    self.logger().debug(f"Got ws message {decode}")
                     if decode.get("error") is not None:
                         self.logger().error(decode["error"])
                         continue
 
-                    if decode.get("content") is not None:
-                        content_type = decode["event_type"]
-
+                    content_type = decode.get("event_type")
+                    if content_type is not None:
                         if content_type in ["balance", "order"]:  # balance: Balance Delta, order: Order Delta
                             output.put_nowait(decode)
                         elif content_type == "re-authenticate":
                             await self.authenticate()
                         elif content_type == "heartbeat":
+                            self.logger().debug("WS heartbeat")
```

Comment:
This seems as a log introduced only for the issue investigation

Reply:
ditto